AdminWebsite/AdminWebsite/ClientApp/src/app/common/constants.ts

```diff
@@ -44,5 +44,5 @@ export const Constants = {
     DefenceAdvocate: 'App Advocate',
     None: 'None',
     RespondentAdvocate: 'Respondent Advocate',
-    JudiciaryRoles: ['Panel Member', 'Winger']
+    JudiciaryRoles: []
```

Comment:
Hi Prasanna, there is impact of this constant in these components, did you get a chance to check them?

src\app\services\video-hearings.service.ts
src\app\services\search.service.ts
src\app\booking\search-email\search-email.component.ts
src\app\booking\add-participant\add-participant.component.ts

Reply:
or am I mistaken that these roles or checks are not required for 1.27?